dev-qt/qt-creator/qt-creator-9999.ebuild

```diff
@@ -66,12 +66,7 @@ CDEPEND="
 	>=kde-frameworks/syntax-highlighting-5.87:5
 	clang? (
 		>=dev-cpp/yaml-cpp-0.6.2:=
-		|| (
-			sys-devel/clang:13
-			sys-devel/clang:12
-			sys-devel/clang:11
-		)
-		<sys-devel/clang-$((LLVM_MAX_SLOT + 1)):=
+		sys-devel/clang:13=
```

Comment:
why bother touching this if you're removing the ebuild in the next commit?

Reply:
My mistake.  Didn't mean to copy over 9999.